packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-card.vue

```diff
@@ -24,65 +24,103 @@
+				Starting at day
+				<tera-input-number
+					auto-width
+					:model-value="intervention.staticInterventions[0].timestep"
+					@update:model-value="(val) => onUpdateThreshold(val, 0)"
+					placeholder="timestep"
+				/>
+				,
+			</div>
 			<div class="flex align-items-center flex-wrap gap-2">
-				Set
-				<section>
-					<Dropdown
-						class="type-menu"
-						:model-value="intervention.type"
-						@change="onSemanticChange"
-						:options="interventionSemanticOptions"
-						option-label="label"
-						option-value="value"
-					/>
-					<Dropdown
-						class="applied-to-menu"
-						:model-value="intervention.appliedTo"
-						@change="onAppliedToParameterChange"
-						:options="semanticOptions"
-						option-label="label"
-						option-value="value"
-						placeholder="Select"
-					/>
-				</section>
+				<template v-if="interventionType === 'dynamic'">
+					Set
+					<section>
+						<Dropdown
+							class="type-menu"
+							:model-value="intervention.dynamicInterventions[0].type"
+							@change="onSemanticChange($event, 0)"
+							:options="interventionSemanticOptions"
+							option-label="label"
+							option-value="value"
+						/>
+						<Dropdown
+							class="applied-to-menu"
+							:model-value="intervention.dynamicInterventions[0].appliedTo"
+							@change="onAppliedToParameterChange($event, 0)"
+							:options="semanticOptions"
+							option-label="label"
+							option-value="value"
+							placeholder="Select"
+						/>
+					</section>
+				</template>
 				<!-- Static -->
 				<template v-if="interventionType === 'static'">
-					to
+					<!-- to -->
 					<template v-if="intervention.staticInterventions.length > 1">...</template>
 					<template v-else-if="intervention.staticInterventions.length === 1">
+						Set
+						<section>
+							<Dropdown
+								class="type-menu"
+								:model-value="intervention.staticInterventions[0].type"
+								@change="onSemanticChange($event, 0)"
+								:options="interventionSemanticOptions"
+								option-label="label"
+								option-value="value"
+							/>
+							<Dropdown
+								class="applied-to-menu"
+								:model-value="intervention.staticInterventions[0].appliedTo"
+								@change="onAppliedToParameterChange($event, 0)"
+								:options="semanticOptions"
+								option-label="label"
+								option-value="value"
+								placeholder="Select"
+							/>
+						</section>
+						to
 						<tera-input-number
 							auto-width
 							:model-value="intervention.staticInterventions[0].value"
 							@update:model-value="(val) => onUpdateValue(val, 0)"
 							placeholder="value"
 						/>
-						starting at
-						<tera-input-number
-							auto-width
-							:model-value="intervention.staticInterventions[0].timestep"
-							@update:model-value="(val) => onUpdateThreshold(val, 0)"
-							placeholder="timestep"
-						/>
-						.
 					</template>
 
 					<ul v-if="intervention.staticInterventions.length > 1" class="w-full">
```

Comment:
I believe this should be an `v-else-if` to avoid the `...` in this scenario
![image](https://github.com/user-attachments/assets/b8a71dce-92ff-4f98-8bf3-0b0c31c68299)